tests/test_features/test_scatter.py

```diff
@@ -65,6 +66,10 @@ def test_init_alias(self):
         visualizer = ScatterVisualizer(features=features, markers=['*'])
         self.assertIsNotNone(visualizer.markers)
 
+    def test_depreciated(self):
+        features = ["temperature", "relative_humidity"]
+        assert_warns_message(DeprecationWarning, "Will be moved to yellowbrick.contrib in v0.7", ScatterVisualizer, features=features)
```

Comment:
Awesome! I think deprecation warning tests are vital for future us to remove them!